otftotfm/otftotfm.cc

```diff
@@ -63,6 +63,9 @@
 #ifdef HAVE_FCNTL_H
 # include <fcntl.h>
 #endif
+#ifdef _MSC_VER
+# include <io.h>
+#endif
```

Comment:
It turned out that Luigi needed this for MinGW as well (as I suspected), so it made sense to put it under `#ifdef WIN32`. I probably didn't need it since I disabled kpathsea, and it was probably needed for parts ifdef-ed for kpathsea functionality.